packages/mobile/src/components/feed-tip-tile/SendTipButton.tsx

```diff
@@ -70,7 +71,14 @@ export const SendTipButton = (props: SendTipButtonProps) => {
   return (
     <View style={styles.sendTipButton}>
       <Button
-        title={
+        onPress={handlePress}
+        onPressIn={handlePressIn}
+        onPressOut={handlePressOut}
```

Comment:
not necessary to change now, but if this is a common pattern, maybe button could pass this as a prop to its children formik style so you could do
```
<Button>
{ ({isActive}) => <Text style={isActive ? ... }>...</Text> }
</Button>
```

Reply:
yeah great idea, will look into. personally i care not at all for this component because it's only present when monetization is disabled. we should probably just remove this whole feature. will bring it up in the channel when design is back.